v2/yarpcprotobuf/codec.go

```diff
@@ -81,3 +122,13 @@ func (c *jsonCodec) Encode(res interface{}) (*yarpc.Buffer, error) {
 	}
 	return nil, yarpcerror.InternalErrorf("tried to encode a non-proto.Message in json codec")
 }
+
+func (c *jsonCodec) EncodeError(err error) (*yarpc.Buffer, error) {
+	// unclear if we should encode a google/rpc/status.proto here or serialize 
```

Comment:
This seems sensible to me, especially since it's not clear if details are always proto. We may want to note: https://github.com/grpc/grpc-go/issues/2329